### ISSUE TITLE
Better VNC screen resolution

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#766]
**Is your feature request related to a problem? Please describe.**
The default VNC screen resolution for the X desktop is 1900x1200 which exceeds many user's client resolution. This can lead to much scrolling and difficulty when maximising an application.

**Describe the solution you'd like**
It may be advantageous to implement a lower default resolution, e.g. 1024x600. It may further be advantageous to provide a mechanism for changing the resolution. This could be in webconf and/or the desktop itself.

**Additional context**
I set this high resolution to allow a large (virtual) desktop for lots of applications but in practice it has proven irksome.

**Comment by riban-bw** [comment]
The setting for the VNC desktop resolution is `$geometry` in file `/etc/vnc.conf `.

**Comment by jofemodo** [comment]
Perhaps we could try setting the $geometry using the browser's window size. It's not perfect, but it should be better than current behaviour.

**Comment by jofemodo** [comment]
Nobody is complaining about this. I would close.

**Comment by riban-bw** [comment]
I complain every time I use VNC (which is every day)! I raised the issue knowing it was in my gift to resolve but then didn't do anything about it. Indeed it is assigned to me! I might just do some work in the background to improve my experience. The reason I raised the ticket was to highlight the problem and get feedback on preferences but as you say, no one seems particularly engaged so I will just do what I want!!!!
